### PR TITLE
Add compat data for WorkerLocation (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/WorkerLocation

Assume the properties listed will come under `URLUtilsReadOnly`.

### CODE REVIEW

File: api/WorkerLocation.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "api": {
+    "WorkerLocation": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerLocation",
+        "support": {
+          "webview_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerLocation.json]: Change to `true`.

File: api/WorkerLocation.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          },
+          "chrome": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerLocation.json]: Change to `"1"`.

File: api/WorkerLocation.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerLocation.json]: Change to `true`.

File: api/WorkerLocation.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            {
+              "version_added": "29",
+              "notes": "<code>origin</code>"
+            }
+          ],
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
```
Comment (jpmedley) [inline api/WorkerLocation.json]: Change both versions of Opera to `true`.

**Comment by maboa** [comment]
Changes have been made :)

**Pull request merged**